zigpy_xbee/zigbee/application.py

```diff
@@ -385,6 +386,44 @@ def handle_rx(
 
         self.handle_message(device, profile_id, cluster_id, src_ep, dst_ep, data)
 
+    def neighbors_updated(
+        self, ieee: zigpy.types.EUI64, neighbors: list[zdo_t.Neighbor]
+    ) -> None:
+        """Neighbor update from Mgmt_Lqi_req."""
+        for neighbor in neighbors:
+            if neighbor.relationship == zdo_t._NeighborEnums.Relationship.Parent:
```

Comment:
The `_NeighborEnums` type is zigpy-internal. Use `zdo_t.Neighbor.Relationship` instead.

```suggestion
            if neighbor.relationship == zdo_t.Neighbor.Relationship.Parent:
```

Reply:
Thank you! You are correct!